packages/design-systems/src/components/composite/Datagrid/SearchFilter/CustomFilter.test.tsx

```diff
@@ -148,7 +150,9 @@ describe(
             currentFilters = currentState;
           }}
           localization={LOCALIZATION_JA}
-          isVisible={true}
+          customFilterOpen={true}
+          /* eslint-disable-next-line @typescript-eslint/no-empty-function */
+          setCustomFilterOpen={() => {}}
```

Comment:
めちゃくちゃ細かいのですが、ここでdisableしているeslint ruleは以下の書き方で対応できますので、そうしてもらいたいです (他の箇所も同様)

```suggestion
          setCustomFilterOpen={() => void 0}
```

Reply:
なるほど承知しました。対応します。